meta-iot2000-example/recipes-support/swupdate/swupdate/defconfig

```diff
@@ -23,15 +24,16 @@ CONFIG_SOCKET_CTRL_PATH="/tmp/sockinstctrl"
 CONFIG_SOCKET_PROGRESS_PATH="/tmp/swupdateprog"
 CONFIG_SOCKET_REMOTE_HANDLER_DIRECTORY="/tmp/"
 # CONFIG_MTD is not set
-# CONFIG_LUA is not set
+CONFIG_LUA=y
+# CONFIG_LUAPKG is not set
 # CONFIG_FEATURE_SYSLOG is not set
 
 #
 # Build Options
 #
 CONFIG_CROSS_COMPILE=""
 CONFIG_SYSROOT=""
-CONFIG_EXTRA_CFLAGS=""
+CONFIG_EXTRA_CFLAGS="-g"
```

Comment:
What's the debug flag being used for?

Reply:
For debugging :)) Well, it was in our default configuration, we can either keep it or delete it. Not my decision.

Reply:
We should not do this on individual recipes but instead on the image build as a whole, i.e., if you build a debug image, then there should be a `-g`, otherwise not.

Reply:
That's true.